vmware/ngc/NGC-AdapterManager/src/main/java/org/opensds/vmware/ngc/adapters/opensds/RestClient.java

```diff
@@ -198,8 +201,23 @@ JSONObject createVolume(String name, String description, ALLOC_TYPE allocType, l
             throw new Exception(msg);
         }
 
-        logger.info("OpenSDS Volume Created.");
-        return response;
+        int retryCount = 0;
+
+        String volumeId = response.getString("id");
+        JSONObject volume = new JSONObject();
+
+        while(retryCount<10) {
```

Comment:
Doubt: 
a) On what basis we have chosen atmax 100sec wait? 
b) Hopefully caller checks for volume.getString("status").equals("available") as after 10 failed retries, we are not checking

Reply:
The wait time was chosen as 10 sec for 10 times retries. If there are any wait time suggestions from hotpot side, let me know.

Reply:
Not sure of the recommendation from Hotpot.. you can put code comment.. if required, in future we will increase or decrease this time